css/templates/lune/pages/_product.css

```diff
@@ -1,3 +1,139 @@
 /* //////////////////////////////////////////////////////
    PRODUCT
-///////////////////////////////////////////////////////*/
\ No newline at end of file
+///////////////////////////////////////////////////////*/
+
+:root {
+  --hero-title-min-height: 238px;
+}
+
+.product-section-hero {
+  min-height: calc(100vh - var(--navbar-height));
+}
+
+/* Set medium resolution make each section a minimum height of 100vh */
+@media (--breakpoint-md) {
+  .product-section {
+    min-height: calc(100vh - var(--wide-navbar-height));
+  }
+}
+
+/* On some titles the titles are split on multiple lines. Apply a hyphen at the end. */
+.lune-title-has-mobile-hyphen:after {
+  content: '-';
+}
+
+/* //////////////////////////////////////////////////////
+   HERO
+///////////////////////////////////////////////////////*/
+
+.hero-content {
+  letter-spacing: var(--letter-spacing-3);
```

Comment:
All the properties I've used in this selector are not defined by basscss. Font size (h0) will hopefully be available soon